test/nbrowser/gristUtils.ts

```diff
@@ -4190,6 +4190,24 @@ export async function getViewportDimensions(): Promise<WindowDimensions> {
   });
 }
 
+/**
+ * Waits until all text in the given input element is selected and the element has focus.
+ * @param inputSelector Selector for the input element to check selection on.
+ */
+export async function waitForSelection(inputSelector: string) {
```

Comment:
Might it be better to keep in "ColorSelect", since it's rather specific to the needs of that test? If you'd rather expose it here for other purposes, then I think it needs a clearer name, since "selection" is used for several different things in our tests (like cell ranges, or selected menu items). Maybe "waitForInputFullSelection"? It could also accept a WebElement, for better generality (you can pass one to executeScript, e.g. as for `scrollIntoView`)